include/drivers/console/uart_pipe.h

```diff
@@ -35,11 +35,13 @@ typedef u8_t *(*uart_pipe_recv_cb)(u8_t *buf, size_t *off);
  *
  *  This function is used to register new UART application.
  *
+ *  @param uart_pipe_dev_name Name of the uart pipe device.
```

Comment:
I don't think this description and overall naming is clear. This parameter names the UART device to use to initialize the "pipe subsystem". "uart pipe device" is ambiguous: there's no "pipe device" (uart one or otherwise); however, there's uart pipe's device.